README.md

```diff
@@ -287,103 +267,164 @@ in your package and update the path definitions accordingly. Specifications are
-│       │   │   │   ├── config.json
-│       │   │   │   ├── marathon.json.mustache
-│       │   │   │   ├── resource.json
-│       │   │   │   └── package.json
-│       │   │   └── ...
-│       │   └── ...
-│       ├── F
-│       │   ├── foo
-│       │   │   ├── 0
-│       │   │   │   ├── config.json
-│       │   │   │   ├── marathon.json.mustache
-│       │   │   │   ├── resource.json
-│       │   │   │   └── package.json
-│       │   │   └── ...
-│       │   └── ...
-│       └── ...
-└── scripts
-    ├── 1-validate-packages.sh
-    ├── build.sh
-    └── install-git-hooks.sh
-```
+2. Ensure that the `jsonschema` command line tool is installed:
 
-## Sources and Transfer Protocols
+  ```bash
+  sudo pip install jsonschema
+  ```
 
-This section describes transfer of package metadata from a universe source to a client program.
+3. Run the verification and build script:
 
+  ```bash
+  scripts/build.sh
+  ```
+
+4. Verify all build steps completed successfully
+5. Submit a pull request against the `version-3.x` branch with your changes. Every pull request opened will have a set
+   of automated verifications run against it. These automated verification are reported against the pull request using
+   the GitHub status API. All verifications must pass in order for a pull request to be eligible for merge.
+
+6. Respond to manual review feedback provided by the DC/OS Community.
+  * Each Pull Request to Universe will also be manually reviewed by a member of the DC/OS Community. To ensure your
+    package is able to be made available to users as quickly as possible be sure to respond to the feedback provided.
+
+
+## Repository Consumption
+
+In order for Universe to be consumed by DC/OS the build process needs to be run to create the Universe Server.
+
+### Universe Server
+
+Universe Server is a new component introduce alongside `packagingVersion` `3.0`. In order for Universe to be able to
+provide packages for many versions of DC/OS at the same time, it is necessary for a server to be responsible for serving
+the correct set of packages to a cluster based on the cluster's version.
+
+All Pull Requests opened for Universe and the `version-3.x` branch will have their Docker image built and published
+to the DockerHub image [`mesosphere/universe-server`](https://hub.docker.com/r/mesosphere/universe-server/).
+In the artifacts tab of the build results you can find `docker/server/marathon.json` which can be used to run the
+Universe Server for testing in your DC/OS cluster.  For each Pull Request, click the details link of the "Universe Server
+Docker image" status report to view the build results.
+
+#### Build Universe Server locally
+
+1. Validate and build the Universe artifacts
+  ```bash
+  scripts/build.sh
+  ```
+
+2. Build the Universe Server Docker image
+  ```bash
+  DOCKER_TAG="my-package" docker/server/build.bash
+  ```
+  This will create a Docker image `universe-server:my-package` and `target/marathon.json` on your local machine
```

Comment:
Shouldn't this is `docker/target/marathon.json`? I didn't see `target/marathon.json` (although there is a `target/universe.json`).